ui/js/services/main.js

```diff
@@ -57,27 +57,6 @@ treeherder.factory('thUrl', [
 
     }]);
 
-treeherder.factory('ThPaginator', function () {
```

Comment:
Has been unused since it was added in 2014! (66b94a6a0dc93c355846f6f1c417d5e628f66c98)